examples/skorch_simple.py

```diff
@@ -0,0 +1,108 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using skorch.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+skorch, and MNIST. We optimize the neural network architecture. As it is too time
+consuming to use the whole MNIST dataset, we here use a small subset of it.
+
+You can run this example as follows, pruning can be turned on and off with the `--pruning`
+argument.
+    $ python skorch_simple.py [--pruning]
+
+"""
+
+import argparse
+
+import numpy as np
+import optuna
+from optuna.integration import SkorchPruningCallback
```

Comment:
Import statements of `optuna` modules should be placed after the other third-party libraries like [this](https://github.com/optuna/optuna/blob/master/examples/pytorch_simple.py).